### DISCUSSION TITLE
Is there a way to kill current job if a new commit is preset for the PR ?

### DISCUSSION THREAD

**Discussion by jeran7x** [community/community#26452] [Actions]
<p>Currently i am trying to build CI workflow using actions, and we try to build a PR before merging with target branch.</p>
<p>So the problem after creating a PR if we create more than one commit to that branch, actions will build each of them. But is there a way to kill there job for the older commit as it is no longer valid (oudated) ?  </p>

**Comment by mscoutermarsh** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeran7x">@jeran7x</a>,</p>
<p>You could give this Action a try: <a href="https://github.com/marketplace/actions/cancel-workflow-action" rel="nofollow noopener">https://github.com/marketplace/actions/cancel-workflow-action</a></p>
<p>There’s nothing currently built into Actions that will do this for you. But it’s possible via the Actions API (which ^ Action uses).</p>

**Comment by jeran7x** [comment]
<p>This is exactly what I am looking for, TY kind sir</p>
